src/dropdown/list/dropdown-list.component.ts

```diff
@@ -405,10 +406,12 @@ export class DropdownList implements AbstractDropdownView, AfterViewInit, OnDest
 	doKeyDown(event: KeyboardEvent, item: ListItem) {
 		// "Spacebar", "Down", and "Up" are IE specific values
 		if (event.key === "Enter" || event.key === " " || event.key === "Spacebar") {
-			event.preventDefault();
-			if (event.key === "Enter") {
-				this.doClick(event, item);
-			}
+				if (this.listElementList.some(option => option.nativeElement === event.target)) {
+					event.preventDefault();
```

Comment:
Did you check this with an `input` as a custom template? That's where the original issue ran into problems ...

Reply:
Yup I tested it with the search and it was working

Reply:
Awesome, we should be good to go then